configuration/ConfigHandler.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef CONFIG_CONFIGHANDLER_HPP
+#define CONFIG_CONFIGHANDLER_HPP
+
+#include <string>
+#include <map>
+
+namespace Configuration
+{
+
+    /**
+     * @brief Handles the server configuration
+     * by parsing the config.yml file
+     */
+    class ConfigHandler
+    {
+    public:
+        ConfigHandler(std::string);
+        ~ConfigHandler();
+
+        std::string getIP(void);
+        std::string getMotd(void);
+        std::string getPort(void);
```

Comment:
Why is the port stored as an `std::string` and not an `uint16_t` ?

Reply:
Because I'm very bad at cpp, so everything is a std::string to make it easier for me

Reply:
Well now is the time to try and make better cpp :)